content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -581,13 +582,15 @@ Each feature gate is designed for enabling/disabling a specific feature:
   When enabled, kubelet CRI interface and authenticated http servers are instrumented to generate
   OpenTelemetry trace spans.
   See [Traces for Kubernetes System Components](/docs/concepts/cluster-administration/system-traces) for more details.
-- `LegacyServiceAccountTokenCleanUp`: Enable cleaning up Secret-based
+- `LegacyServiceAccountTokenCleanUp`: Enable invalidating auto-generated Secret-based
   [service account tokens](/docs/concepts/security/service-accounts/#get-a-token)
-  when they are not used in a specified time (default to be one year).
+  when they are not used in a specified time (default to be one year). Clean up
+  the auto-generated Secret-based tokens if they have been invalidated for a specified time
+  (default to be one year).
 - `LegacyServiceAccountTokenTracking`: Track usage of Secret-based
   [service account tokens](/docs/concepts/security/service-accounts/#get-a-token).
-- `LoadBalancerIPMode`: Allows setting `ipMode` for Services where `type` is set to `LoadBalancer`. 
-  See [Specifying IPMode of load balancer status](/docs/concepts/services-networking/service/#load-balancer-ip-mode) 
+- `LoadBalancerIPMode`: Allows setting `ipMode` for Services where `type` is set to `LoadBalancer`.
+  See [Specifying IPMode of load balancer status](/docs/concepts/services-networking/service/#load-balancer-ip-mode)
```

Comment:
If you can, avoid making whitespace changes here (and squash down). It helps avoid merge conflicts.

Reply:
Revised.